infinispan/infinispan
# ISPN-9147 Upload wildfly modules to Infinispan download page (+ server branding fixes)

https://issues.jboss.org/browse/ISPN-9147
https://issues.jboss.org/browse/ISPN-9148

To verify ISPN-9147:
`mvn clean install -DskipTests -Pdistribution`
At the end verify that the distribution/target/distribution directory contains the wildfly modules zip

To verify ISPN-9148:
After the build, look at `server/integration/build/target/infinispan-server-9.3.0-SNAPSHOT/modules/system/layers/base/org/jboss/as/product/ispn/dir/META-INF/MANIFEST.MF` and ensure that all properties have been correctly replaced.
